### PR TITLE
arash/RTS28 2019

### PR DESCRIPTION
**Pull request by arash-binary**

### CODE REVIEW

File: src/templates/static/legal/regulation.jsx [hunk 1]
```diff
@@ -162,18 +162,32 @@ const Regulation = () => {
...
+                    <h3>{it.L('RTS 28')}</h3>
+                    <div className='gr-row'>
+                        <div className='gr-6 gr-12-m'>
+                            <FillBox
+                                align_left
+                                center
+                                border='border-dark-gray'
+                                className='margin-top-17 align-start'
+                                image='images/pages/regulation/pdf-icon.svg'
+                                href={it.url_for('/download/BIEL-RTS28.pdf')}
```
Comment (ashkanx) [inline src/templates/static/legal/regulation.jsx]: please make sure file names are consistent as well

Comment (arash-binary) [reply src/templates/static/legal/regulation.jsx]: Sure.

File: src/templates/static/legal/regulation.jsx [hunk 2]
```diff
@@ -162,18 +162,32 @@ const Regulation = () => {
...
+                            />
+                        </div>
+                        <div className='gr-6 gr-12-m'>
+                            <FillBox
+                                align_left
+                                center
+                                border='border-dark-gray'
+                                className='margin-top-17 align-start'
+                                image='images/pages/regulation/pdf-icon.svg'
+                                href={it.url_for('/download/BIEL-RTS28-for-2017.pdf')}
```
Comment (negarn) [inline src/templates/static/legal/regulation.jsx]: should this be 2018?

Comment (arash-binary) [reply src/templates/static/legal/regulation.jsx]: Sorry, this happened when I was changing the file's name.

**Pull request merged**